src/challenge-manager.ts

```diff
@@ -135,7 +136,7 @@ export class ChallengeManager {
     disputedWitness: WitnessProof,
     caller: string
   ): void {
-    if (this.interval() <= 1) {
+    if (this.interval() < 1) {
```

Comment:
I wonder if there is another way we can make this work. It seems wrong to split leaves when the interval is equal to 1.